libsolidity/formal/Why3Translator.cpp

```diff
@@ -757,6 +757,11 @@ bool Why3Translator::visit(Literal const& _literal)
 	return false;
 }
 
+bool Why3Translator::visit(PragmaDirective const&)
+{
+	return false;
```

Comment:
Since we might be adding new pragmas in the future which could change the semantics of the language, please only ignore the "version" pragma.